website/src/components/NewRun/utils.ts

```diff
@@ -0,0 +1,71 @@
+import { LOCAL_FLAGS, LOCAL_OUTPUT } from "../../consts/localForage"
```

Comment:
As retrieving from local storage will be useful for individual visualisations as well, would it be better to put this file in a shared utilities folder?

Reply:
not for now, these utils are reserved for stuff produced by a new run

Reply:
Sounds good